draft-ietf-wimse-arch.md

```diff
@@ -77,17 +77,42 @@ Attestation is the function through which a task verifies the identity of a sepa
 
 # Architecture
 
-## Workload Identity
+## Workload Identity {#whimsical-identity}
 
-Workload identity often comprises multiple attributes that describe various aspects of a workload. These attributes can encompass diverse sets of information, including the workload's role within a system, the software it operates, and the hardware environment it utilizes. Different authorities across various parts of the system define these attribute sets. This architecture introduces a Workload Identifier for representing and referencing workloads, enabling diverse authorities with distinct identity attribute sets to consistently reference the workload.
+The identity of a workload is a complex topic.  The WIMSE architecture document starts by defining a trust domain, a workload identifier and workload identity credentials. These constructs are sufficient for establishing basic authentication, authorization and accounting. More complex identity constructs can be created from these basic building blocks.
 
-The Workload Identifier consists of a concise string allocated within a namespace defined by a Trust Domain. This Workload Identifier is present in Workload Identity Tokens and X.509 certificates issued by the authority for the Trust Domain. The Workload Identifier is used to associate additional identity attributes to the workload through the use of tokens (workload attribute tokens?) or online look up services. It may also be used directly in authorization calculations and audit logs.
+### Trust Domain
 
-The Trust Domain consists of a string that matches the format of a fully qualified domain name. It is the intent that a Trust Domain is actually a domain name registered to the organization defining the Trust Domain, but this may not be true in all cases. The Trust Domain also maps to the issuer of cryptographically signed Workload Identity Tokens (WIT) or X.509 Certificate. The association between a Trust Domain and the cryptographic root of the signing authority for that Trust Domain must be made securely through an out-of-band mechanisms.
+A trust domain is a logical grouping of systems that share a common set of security controls and policies. WIMSE certificates and tokens are issued under the authority of a trust domain. Trust domains SHOULD be identified by a fully qualified domain name belonging to the organization defining the trust domain. A trust domain maps to one or more trust anchors for validating X.509 certificates and a mechanism to securely obtain a JWK Set {{!RFC7517}} for validating WIMSE WIT tokens. This mapping MUST be obtained through a secure mechanism that ensures the authenticity and integrity of the mapping is fresh and not compromised. This secure mechanism is out of scope for this document.
```

Comment:
I would not introduce new "WIMSE certificates" construct unnecessarily. Perhaps we could change it to "Workload certificates"?
Maybe "associated with" is better than "belonging" when it comes to domains?
"WIMSE WIT tokens"? Or "WIMSE tokens"?

Reply:
Changed to workload, and associated. WIT is what is defined in the S2S spec. Keeping as WIMSE WIT for now.